lib/matplotlib/axes/_axes.py

```diff
@@ -2392,27 +2392,78 @@ def broken_barh(self, xranges, yrange, **kwargs):
     @_preprocess_data(replace_all_args=True, label_namer=None)
     def stem(self, *args, **kwargs):
         """
+        stem(*args, linefmt=None, markerfmt=None, basefmt=None, data=None,
```

Comment:
Ah, but of course that doesn't work with the doc generation in plt.plotting which tries to parse the first line of the docstring for a summary table (see build failure).

Long term plan should be to fix plt.plotting but in the meantime for this specific case you can hopefully get away with moving stuff into the actual signature...